### ISSUE TITLE
Update default loss function

### ISSUE THREAD

**Issue by PierreGtch** [braindecode/braindecode#521]
we will need to update the default loss function `EEGClassifier` to `nn.CrossEntropyLoss` because of #513  
Also, we will need to change the methods `predict`, `predict_probas`, and `predict_log_probas` .
Finally, we need to document that `EEGClassifier` expects models that don’t have a softmax/logsoftmax layer.
